sopel/bot.py

```diff
@@ -376,6 +376,15 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, nick, channel, text=None):
+        """Send an IRC KICK command.
+
+        Within the context of a triggered callable, ``channel`` will default to the
+        channel in which the call was triggered. If triggered from a private message,
+        ``channel`` is required (or the call to ``kick()`` will be ignored).
```

Comment:
Accepting this part of the suggested docstring means that if triggered in privmsg, outside a channel, `SopelWrapper.kick()` should return without doing anything if not given a channel name.